src/osgEarthDrivers/label_mp_screen_annotations/MPAnnotationLabelSource.cpp

```diff
@@ -81,10 +81,12 @@ class MPAnnotationLabelSource : public LabelSource
         NumericExpression textRotationExpr( text ? *text->onScreenRotation() : NumericExpression() );
         NumericExpression textCourseExpr  ( text ? *text->geographicCourse() : NumericExpression() );
         StringExpression  textOffsetSupportExpr ( text ? *text->autoOffsetGeomWKT()  : StringExpression() );
+        NumericExpression textPriorityFineExpr ( text ? *text->textPriority()  : NumericExpression() );
```

Comment:
**Impossible to understand the difference between TextSymbol::priority() and TextSymbol::textPriority**

replace textPriority by priorityFine everywhere